README.md

```diff
@@ -59,22 +63,34 @@ Fastly supports Plotly.js with free CDN service. Read more at <https://www.fastl
     <div id="gd"></div>
 
     <script>
-        Plotly.newPlot("gd", {
-            "data": [{
-                "y": [1, 2, 3]
-            }],
-            "layout": {
-                "width": 600,
-                "height": 400
-            }
-        });
+        Plotly.newPlot("gd", /* JSON object */ {
+            "data": [{ "y": [1, 2, 3] }],
+            "layout": { "width": 600, "height": 400}
+        })
     </script>
 </body>
 ```
-In the example above `Plotly` object is added to the window scope by the script in the `head` section.
-The `newPlot` method is then used to draw an interactive figure as described by `data` and `layout` into the desired `div` here named `gd`.
-As demonstrated in the example above basic knowledge of `html` and [JSON](https://en.wikipedia.org/wiki/JSON) syntax is enough to get started i.e. with/without JavaScript!
-To learn and build more with plotly.js please visit [plotly.js documentation](https://plotly.com/javascript).
+
+### Native ES6 JavaScript import
```

Comment:
This is probably redundant for now... anyone that knows this technique will see the bit above it and know it's possible.

Reply:
(I worry about this "loading" section just being too long basically)